client/collator/src/lib.rs

```diff
@@ -415,10 +417,14 @@ mod tests {
 			_: PHash,
 			validation_data: &PersistedValidationData,
 		) -> Option<ParachainCandidate<Block>> {
+			let mut sproof = RelayStateSproofBuilder::default();
+			sproof.included_para_head = Some(HeadData(parent.encode()));
+			sproof.para_id = cumulus_test_runtime::PARACHAIN_ID.into();
+
```

Comment:
I believe there's more suitable place for this code
https://github.com/paritytech/cumulus/blob/b67b0d6bbc310b8e84d9981bf228b35fc1e8552d/test/client/src/block_builder.rs#L67-L73

Reply:
That loses generality, e.g. setting the included para head to _not_ be in there.